production/helm/meta-monitoring/values.yaml

```diff
@@ -0,0 +1,165 @@
+integrations:
+  collector: alloy-singleton
+  alloy:
+    instances:
+      # monitor the collectors gathering and sending metrics/logs to the local cluster
+      - name: alloy
+        labelSelectors:
+          app.kubernetes.io/name: [alloy-singleton]
+        namespaces:
+          - meta
+
+  loki:
+    instances:
+      - name: loki
+        namespaces:
+          - loki
+        labelSelectors:
+          app.kubernetes.io/name: loki
+        logs:
+          tuning:
+            # extract logfmt fields and set them as structured metadata
+            structuredMetadata:
+              caller:
+              tenant:
+              org_id:
+              user:
+# (Optional) Kubernetes Events are captured as logs and are annotated with additional metadata to make them easier to search and filter.
+clusterEvents:
+  enabled: true
+  collector: alloy-singleton
+  namespaces:
+    - meta
+    - loki
+# A collection of metric collectors that gather metrics from various sources in the cluster.
+# (Required) cadvisor - Used to collect Loki pod metrics. Cadvisor is automatically deployed.
+# kubelet - Kubernetes information on each node
+# kubeletResource - Scrape resource metrics from the Kubelet.
+# kube-state-metrics - A simple service that listens to the Kubernetes API server and generates metrics about the state of the objects.
+clusterMetrics:
+  enabled: true
+  collector: alloy-singleton
+  kubelet:
+    enabled: true
+  kubeletResource:
+    enabled: true
+  cadvisor:
+    enabled: true
+    extraMetricProcessingRules: |-
+      rule {
+        action = "keep"
+        source_labels = ["namespace"]
+        regex = "meta|loki"
+      }
+  apiServer:
+    enabled: false
+  kubeControllerManager:
+    enabled: false
+  kubeDNS:
+    enabled: false
+  kubeProxy:
+    enabled: false
+  kubeScheduler:
+    enabled: false
+  kube-state-metrics:
+    enabled: true
+    namespaces: loki,meta
+    metricsTuning:
+      useDefaultAllowList: false
+      includeMetrics: [(.+)]
+  node-exporter:
+    enabled: true
+    deploy: true
+    metricsTuning:
+      useIntegrationAllowList: true
+  windows-exporter:
+    enabled: false
+    deploy: false
+  kepler:
+    enabled: false
+    deploy: false
+  opencost:
+    enabled: false
+    deploy: false
+
+nodeLogs:
+  enabled: false
+
+# Enable pod log collection for the cluster. Will collect logs from all pods in both the meta and loki namespace. 
+podLogs:
+  enabled: true
+  collector: alloy-singleton
+  labelsToKeep:
+    - app
+    - app_kubernetes_io_name
+    - component
+    - container
+    - job
+    - level
+    - namespace
+    - pod
+    - service_name
+    - cluster
+  gatherMethod: kubernetesApi
+  namespaces:
+    - meta
+    - loki
+
+# Collectors
+
+# The Alloy Singleton is a single instance of the Alloy Collector that is deployed in the cluster.
+alloy-singleton:
+  enabled: true
+
+alloy-metrics:
+  enabled: false
+
+alloy-logs:
```

Comment:
Curious as to why alloy-logs is false.

Reply:
Good question, since we are only meta-monitoring Loki we just want a a small footprint Alloy rather than a cluster. Alloy singleton will do that job